hr_timesheet_overtime/views/resource_view.xml

```diff
@@ -36,7 +36,7 @@
         <field name="name">resource.overtime.tree</field>
         <field name="model">resource.overtime</field>
         <field name="arch" type="xml">
-            <tree string="Overtime Rate">
+            <tree>
```

Comment:
This was done in the prettier/black/isort commit. Is this intentional?

Reply:
Yes, tree names are not displayed and deprecated : https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#views